doc/source/examples/sharp-immersed-boundary-solver/sedimentation-64-particles/sedimentation-64-particles.rst

```diff
@@ -0,0 +1,349 @@
+ ``/examples/sharp-immersed-boundary-solver/sedimentation-64-particle/sedimentation-64-particle.prm``
+ 
+ ``/examples/sharp-immersed-boundary-solver/sedimentation-64-particle/particles.input``
+
+Description of the case
+-----------------------
+The case consists in the release of 64 particles (:math:`\rho_p=0.0015 \frac{\text{kg}}{\text{cm}^{3}}`) with a diameter of 0.25 cm arranged in a 4 by 4 by 4 cubic array centered 21 cm above the bottom of the container. The container is a 2 by 2 by 24 cm rectangle. The viscosity of the fluid is :math:`\mu_f=0.0001 \frac{\text{kg}}{\text{s cm}}`. The density of the fluid is :math:`\rho_f=0.001 \frac{\text{kg}}{\text{cm}^{3}}`. The gravity constant is :math:`g= -981 \frac{\text{cm}}{\text{s}^{2}}`. The particles accelerate due to gravity until they hit the bottom of the container, which at this point, we stop the simulation. All the container walls have no-slip boundary conditions except at the top of the container, where we define an open boundary.
+
+Parameter file
+---------------
+
+We explain every part of this parameter file in detail. In each section of the parameter file, we describe relevant parameters. The omitted parameters are only user preference parameters and have no impact on the simulation results. For more information on these parameters we suggest visiting the :doc:`../../../parameters/parameters`.
+ 
+Simulation and IO control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+    subsection simulation control
+      set method             = bdf2
+      set bdf startup method = multiple step bdf
+      set time step          = 0.0025 # Time step
+      set time end           = 5      # End time of simulation
+      set output name        = out    # Prefix for VTU outputs
+      set output frequency   = 1      # Frequency of simulation output
+      set subdivision        = 1      # Mesh subdivision when outputend
+    end
+
+
+* The ``method`` is set to  ``bdf2`` to have a second-order time-stepping method. This ensures a low error due to the time discretization in this case.
+
+* The ``bdf startup method`` is set to  ``multiple step bdf``  as we do not have an initial solution that allows us to generate previous time steps. The ``multiple step bdf`` approach will ramp the order of the scheme in the first few time steps.
+
+* The ``time step`` is set to  0.0025. This time step is small enough to prevent large error due to the time discretization. 
+
+* The ``time end`` is set to  4.0. This is slightly longer than the time needed for all the particles to reach the bottom of the container
+
+
+
+
+Physical properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.1
+        set density             = 0.001
+      end
+    end
+
+
+* The ``kinematic viscosity`` is set to  0.1. This value is derived from the case description by dividing :math:`\mu_f` by :math:`\rho_f`.
```

Comment:
```suggestion
* The ``kinematic viscosity`` is set to  0.1. This value is derived from the case description by dividing :math:`\mu_f` by :math:`\rho_f`. This parameter changes the Reynolds number of the case since it is one of the variables in the evaluation of the Reynolds number but also by changing the terminal velocity of the particle.
```